napari/layers/graph/graph.py

```diff
@@ -399,10 +401,27 @@ def add(
         coords : sequence of indices to add point at
         indices : optional indices of the newly inserted nodes.
         """
+        self.events.data(
+        value=self.data,
+        action=ActionType.ADDING,
+        data_indices=tuple(
+                self.selected_data,
+            ),
+            vertex_indices=((),),
+        )
+
         prev_size = self.data.n_allocated_nodes
         self.data.add_nodes(indices=indices, coords=coords)
         self._data_changed(prev_size)
 
+        self.events.data(
+            value=self.data,
+            action=ActionType.ADDED,
+            data_indices=tuple(
+                    self.selected_data,
+                ),
```

Comment:
We're missing `vertex_indices` here. We want all events to have the same attributes so we should add it back empty (just like above)